lib/src/rules/invariant_booleans.dart

```diff
@@ -149,16 +149,16 @@ Set<Expression> _findConditionsUnderStatementBranch(Statement statement,
 }
 
 TestedExpressions _findPreviousTestedExpressions(Expression node) {
-  Block block = node.getAncestor((a) => a is Block && a.parent is FunctionBody);
-  Iterable<AstNode> nodesInDFS = DartTypeUtilities.traverseNodesInDFS(block,
+  final Block block =
+      node.getAncestor((a) => a is Block && a.parent is FunctionBody);
+  final nodesInDFS = DartTypeUtilities.traverseNodesInDFS(block,
```

Comment:
Similarly, here (and below).  Notice how the `Block` type is intact.  Is that intentional?